packages/cli/src/utils.ts

```diff
@@ -21,6 +21,14 @@ export async function valueOrPrompt<T>(value: T, msg: string, error: string): Pr
   }
 }
 
+export function addV(str: string | undefined) {
+  if (!str?.includes('v')) {
+    const res = 'v';
+    return res.concat(str);
```

Comment:
String interpolation can be used here
```suggestion
    return `v${str}`
```